### PR TITLE
cleanup envs/test

### PR DESCRIPTION
**Pull request by brianhu2006**
This PR make the following changes

- remove heat-stack.yml generation in test/setup

- mv envs/test to envs/example/test. So envs/example/defaults-2.0.yml can be used by  ursula

- cleanup  playbooks/$CI/templates/heat-satck.yml

-  cleanup  playbooks/$CI/vars/main.yml

### CODE REVIEW

**Comment by nirajdp76** [comment]
is this still a WIP?

**Comment by brianhu2006** [comment]
@nirajdp76 :it is not WIP.

By they way, want to make some change on security group but depend on this PR--https://github.com/blueboxgroup/ursula-cli/pull/69

File: test/setup [hunk 1]
```diff
@@ -26,45 +26,29 @@ echo "testenv_instance_prefix: ${testenv_instance_prefix}"
...
+cat <<eof >>  ${ENV_DIR}/group_vars/all.yml
 testenv_instance_prefix: ${testenv_instance_prefix}
 testenv_keypair_name: ${KEY_NAME}
 testenv_keypair_path: ${KEY_PATH}
 eof
 
-cp -r ${ROOT}/envs/example/${TEST_ENV}/* ${ROOT}/envs/test
-echo "creating heat template for test env if no heat_stack.yml in envs/test/..."
-if [[ ! -e ${ROOT}/envs/test/heat_stack.yml ]]; then
-  ursula --ursula-debug "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/create_heat_template.yml --extra-vars="use_private_ips=${USE_PRIVATE_IPS}"
```
Comment (ivy275) [inline test/setup]: If you want to delete these lines, please make sure the heat_stack.yml of all test_envs is under envs/example/<test_env>/ not playbooks/<test_env>/. Such as "allinone"....

Comment (brianhu2006) [inline test/setup]: allinone uses vagrant

File: test/setup [hunk 2]
```diff
@@ -26,45 +26,29 @@ echo "testenv_instance_prefix: ${testenv_instance_prefix}"
...
-eof
-
 echo "booting ${VMS} by heat, then copy files for pre-deployment"
-#sleep 30
 extra_args="--ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name} --ursula-user=${LOGIN_USER} --heat-parameters ${EXTRA_HEAT_PARAMS}"
-ursula ${extra_args} "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@" -vvvv
-mv envs/test/.ssh_config  envs/test/ssh_config
+ursula ${extra_args} "${ENV_DIR}" ${ROOT}/playbooks/${TEST_ENV}/tasks/pre-deploy.yml -u ${LOGIN_USER} "$@" -vvvv
+
+#.ssh_config is only used heat provider
```
Comment (epwn) [inline test/setup]: this comment is not grammatically correct, please fix it

**Comment by epwn** [comment]
LGTM

**Pull request merged**